<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MshFileGeneration</class>
 <widget class="QDialog" name="MshFileGeneration">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>541</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Mesh File Generation</string>
  </property>
  <widget class="Q3ButtonGroup" name="shapeFileGroup">
   <property name="geometry">
    <rect>
     <x>13</x>
     <y>4</y>
     <width>572</width>
     <height>291</height>
    </rect>
   </property>
   <property name="title">
    <string>Input Files</string>
   </property>
   <widget class="QLabel" name="nodeFileLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>57</y>
      <width>72</width>
      <height>38</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="text">
     <string>Node File</string>
    </property>
   </widget>
   <widget class="QLabel" name="eleFileLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>22</y>
      <width>94</width>
      <height>38</height>
     </rect>
    </property>
    <property name="text">
     <string>Element File</string>
    </property>
   </widget>
   <widget class="QLabel" name="neighFileLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>98</y>
      <width>100</width>
      <height>38</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="text">
     <string>Neighbor File</string>
    </property>
   </widget>
   <widget class="QLabel" name="nodeFileLabel_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>169</y>
      <width>131</width>
      <height>38</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="text">
     <string>Surface Elevation</string>
    </property>
   </widget>
   <widget class="QPushButton" name="eleFileButton">
    <property name="geometry">
     <rect>
      <x>469</x>
      <y>27</y>
      <width>98</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Browse...</string>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="nodeFileButton">
    <property name="geometry">
     <rect>
      <x>469</x>
      <y>64</y>
      <width>98</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Browse...</string>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="neighFileButton">
    <property name="geometry">
     <rect>
      <x>469</x>
      <y>102</y>
      <width>98</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Browse...</string>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="sElevFileButton">
    <property name="geometry">
     <rect>
      <x>469</x>
      <y>174</y>
      <width>98</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Browse...</string>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="eleFileLineEdit">
    <property name="geometry">
     <rect>
      <x>162</x>
      <y>32</y>
      <width>294</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="nodeFileLineEdit">
    <property name="geometry">
     <rect>
      <x>162</x>
      <y>68</y>
      <width>294</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="neighFileLineEdit">
    <property name="geometry">
     <rect>
      <x>162</x>
      <y>106</y>
      <width>294</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="sElevFileLineEdit">
    <property name="geometry">
     <rect>
      <x>162</x>
      <y>178</y>
      <width>294</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>210</y>
      <width>551</width>
      <height>71</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLineEdit" name="bElevFileLineEdit">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>153</x>
       <y>14</y>
       <width>294</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>45</y>
       <width>121</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Select Input Type</string>
     </property>
    </widget>
    <widget class="QPushButton" name="bElevFileButton">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>450</x>
       <y>12</y>
       <width>98</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>Browse...</string>
     </property>
     <property name="autoDefault">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QLabel" name="neighFileLabel_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>12</y>
       <width>134</width>
       <height>38</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="text">
      <string>Bed Depth</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit">
     <property name="geometry">
      <rect>
       <x>153</x>
       <y>14</y>
       <width>113</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>5</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButtonUni">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>43</y>
       <width>151</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Uniform Depth (m)</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButtonFile">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>43</y>
       <width>121</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Elevation File</string>
     </property>
    </widget>
   </widget>
   <widget class="QLineEdit" name="rivFileLineEdit">
    <property name="geometry">
     <rect>
      <x>162</x>
      <y>144</y>
      <width>294</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="nodeFileLabel_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>135</y>
      <width>131</width>
      <height>38</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="text">
     <string>River File</string>
    </property>
   </widget>
   <widget class="QPushButton" name="rivFileButton">
    <property name="geometry">
     <rect>
      <x>469</x>
      <y>140</y>
      <width>98</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Browse...</string>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
   </widget>
  </widget>
  <widget class="Q3TextBrowser" name="MessageLog" native="true">
   <property name="geometry">
    <rect>
     <x>13</x>
     <y>403</y>
     <width>570</width>
     <height>130</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
  </widget>
  <widget class="Q3ButtonGroup" name="shapeFileNameGroup">
   <property name="geometry">
    <rect>
     <x>13</x>
     <y>301</y>
     <width>572</width>
     <height>60</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QLabel" name="mshFileLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>9</y>
      <width>135</width>
      <height>38</height>
     </rect>
    </property>
    <property name="text">
     <string>Output Mesh File</string>
    </property>
   </widget>
   <widget class="QPushButton" name="mshFileButton">
    <property name="geometry">
     <rect>
      <x>467</x>
      <y>14</y>
      <width>98</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Browse...</string>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
    <property name="default">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="mshFileLineEdit">
    <property name="geometry">
     <rect>
      <x>161</x>
      <y>16</y>
      <width>294</width>
      <height>25</height>
     </rect>
    </property>
    <property name="cursorPosition">
     <number>0</number>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="runButton">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>365</y>
     <width>98</width>
     <height>34</height>
    </rect>
   </property>
   <property name="text">
    <string>Run!</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelButton">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>365</y>
     <width>98</width>
     <height>34</height>
    </rect>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="helpButton">
   <property name="geometry">
    <rect>
     <x>13</x>
     <y>365</y>
     <width>98</width>
     <height>34</height>
    </rect>
   </property>
   <property name="text">
    <string>Help</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
 <customwidgets>
  <customwidget>
   <class>Q3TextBrowser</class>
   <extends>QWidget</extends>
   <header>q3textbrowser.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Q3GroupBox</class>
   <extends>QGroupBox</extends>
   <header>Qt3Support/Q3GroupBox</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Q3ButtonGroup</class>
   <extends>Q3GroupBox</extends>
   <header>Qt3Support/Q3ButtonGroup</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>radioButtonFile</sender>
   <signal>toggled(bool)</signal>
   <receiver>bElevFileLineEdit</receiver>
   <slot>setShown(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>397</x>
     <y>259</y>
    </hint>
    <hint type="destinationlabel">
     <x>347</x>
     <y>243</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButtonFile</sender>
   <signal>toggled(bool)</signal>
   <receiver>bElevFileButton</receiver>
   <slot>setShown(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>460</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>523</x>
     <y>248</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButtonUni</sender>
   <signal>toggled(bool)</signal>
   <receiver>lineEdit</receiver>
   <slot>setShown(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>285</x>
     <y>265</y>
    </hint>
    <hint type="destinationlabel">
     <x>239</x>
     <y>241</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
